extensions/wikia/Recirculation/js/utils.js

```diff
@@ -65,7 +95,7 @@ define('ext.wikia.recirculation.utils', [
 	 */
 	function loadTemplate(templateName) {
 		var dfd = new $.Deferred(),
-			templateLocation = 'extensions/wikia/Recirculation/templates/client/' + templateName,
+			templateLocation = 'extensions/wikia/Recirculation/templates/' + templateName,
```

Comment:
Ah. Nvm. I see the call below to `utils.renderTemplate('client/footer.mustache', data)`.